content/en/continuous_integration/tests/junit_upload.md

```diff
@@ -322,9 +329,109 @@ For mobile apps (Swift, Android):
 **Examples**: `iPhone 12 Pro Simulator`, `iPhone 13 (QA team)`
 
 
+## Providing metadata with XPath expressions
+
+In addition to the `--tags` CLI parameter and the `DD_TAGS` environment variable, which apply custom tags globally to all tests included the uploaded XML report, the `--xpath-tag` parameter provides custom rules to add tags from different attributes within the XML to each test.
+
+The parameter provided must have the format `key=expression`, where `key` is the name of the custom tag to be added and `expression` is a valid [XPath][10] expression within the ones supported.
+
+XPath syntax is used for familiarity but only a limited subset of expressions are supported:
```

Comment:
```suggestion
While XPath syntax is used for familiarity, but only the following expressions are supported:
```

Reply:
With your suggestion we also need to remove "but" after the comma, right?
```
While XPath syntax is used for familiarity, only the following expressions are supported
```

Reply:
yes, good catch!